DefaultTabBar.js

```diff
@@ -45,20 +47,41 @@ var DefaultTabBar = React.createClass({
     var isTabActive = this.props.activeTab === page;
     var activeTextColor = this.props.activeTextColor || "navy";
     var inactiveTextColor = this.props.inactiveTextColor || "black";
-    return (
-      <TouchableOpacity
-        key={name}
-        accessible={true}
-        accessibilityLabel={name}
-        accessibilityTraits='button'
-        style={[styles.tab]}
-        onPress={() => this.props.goToPage(page)}>
-        <View>
-          <Text style={{color: isTabActive ? activeTextColor : inactiveTextColor,
-            fontWeight: isTabActive ? 'bold' : 'normal'}}>{name}</Text>
-        </View>
-      </TouchableOpacity>
+
+    var innerText = (
+      <Text style={{color: isTabActive ? activeTextColor : inactiveTextColor,
+        fontWeight: isTabActive ? 'bold' : 'normal'}}>{name}</Text>
     );
+
+    if (Platform.OS !== 'android') {
+      return (
+        <TouchableOpacity
+          key={name}
+          accessible={true}
+          accessibilityLabel={name}
+          accessibilityTraits='button'
```

Comment:
https://facebook.github.io/react-native/docs/view.html#accessibilitylabel is not listed as iOS only.